packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -589,10 +600,10 @@ export interface CallClientState {
     deviceManager: DeviceManagerState;
     environmentInfo?: EnvironmentInfo;
     incomingCalls: {
-        [key: string]: IncomingCallState;
+        [key: string]: IncomingCallState | /* @conditional-compile-remove(one-to-n-calling) */ TeamsIncomingCallState;
```

Comment:
This seems like a breaking change...  Is it possible to align TeamsIncomingCallState and IncomingCallState into one class?

Reply:
What's the difference between the two? (Don't suppose IncomingCallState is a complete subset of TeamsIncomingCallState to allow this change?)

Reply:
Oo OK technically not breaking! As the differing properties are new:
![image](https://github.com/user-attachments/assets/b39dac9c-0783-4c0e-8f14-20e24cae0841)

Reply:
To flatten `kind` and `info` into our own `CallInfoState` type to avoid this "breaking" change and any in the future.

Reply:
Just discussed this with Donald - @JamesBurnside you are right! It is not breaking change since we are just adding new properties. (And those properties could be a potential breaking change point in the future)